src/app/api/brands/[brandId]/route.ts

```diff
@@ -0,0 +1,28 @@
+import { response200, response404, response500 } from "@/app/utils";
+import { getBrand } from "@/model/BrandModel";
+import { getProducts } from "@/model/ProductModel";
+
+export async function GET(
```

Comment:
If we're not using this route, let's not add it now.

I don't see any `BrandAPI` so...